core/common/src/main/java/alluxio/wire/FileInfo.java

```diff
@@ -298,6 +299,13 @@ public DefaultAccessControlList getDefaultAcl() {
     return (mDefaultAcl == null) ? new ArrayList<>() : mDefaultAcl.toStringEntries();
   }
 
+  /**
+   * @return a list of pinned locations
+   */
+  public List<String> getPinnedLocation() {
```

Comment:
Does `Location` differ from `Medium`? If not, I would prefer to use the same name